src/components/forms/SignUpForm.js

```diff
@@ -62,7 +62,7 @@ export const SignUpForm = ({ onSuccess, onFailure }) => (
     }}
   >
     {({ errors, touched, isSubmitting }) => (
-      <Form className="flex flex-col gap-4">
+      <Form className="flex flex-col gap-4" data-testid="SignUpForm">
```

Comment:
nit: I think html props usually follow kebab-case so I would call it "sign-up-form" which would feel more natural for me but if you feel good about different casing convention then let me know

example for kebab-case in a different library https://testing-library.com/docs/queries/bytestid/

Reply:
Prop names in general follow a naming convention, but I never heard of a convention about values of HTML attributes (unless we're talking `id` or `class`, which are restricted in terms of allowed characters).

I think it's easier to have a convention where the main renderable element of the component has the same `data-testid` as the component's name.
If we find ourselves adding it on nested elements (e.g. an `input` deeper in `SignUpForm`), I'd consider using a different casing (although I always used pascal case and it worked great for me) :)